source/bin/pr-log.ts

```diff
@@ -47,20 +49,24 @@ program
     .storeOptionsAsProperties(false)
     .description(config.description ?? '')
     .version(config.version ?? '')
-    .arguments('<version-number>')
-    .option('--sloppy', 'skip ensuring clean local git state')
-    .option('--trace', 'show stack traces for any error')
+    .argument('[version-number]', 'Desired version number. Optional when --unreleased was provided')
```

Comment:
The description is confusing. I thought `version-number` must not be provided when `--unreleased` is set.

Reply:
True. The description was a leftover from a very early stage of this pull request 🙈 

Changed in cee04857ee7a1db5bfc368bcfc05ae7c0db06d35